audit/projects/k8s-cip-test-prod/buckets/k8s-cip-test-prod/iam.json

```diff
@@ -4,7 +4,8 @@
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
         "projectEditor:k8s-cip-test-prod",
-        "projectOwner:k8s-cip-test-prod"
+        "projectOwner:k8s-cip-test-prod",
+        "serviceAccount:k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com"
```

Comment:
@puerco I don't remember a change was made for this SA in https://git.k8s.io/k8s.io